pkg/skaffold/schema/v1beta7/upgrade.go

```diff
@@ -26,7 +26,6 @@ import (
 // Upgrade upgrades a configuration to the next version.
 // Config changes from v1beta7 to v1beta8
 // 1. Additions:
-// kaniko/resource requirements
```

Comment:
should this be removed?

Reply:
Of course not, so lazy of me :man_facepalming: